internal/api/installation.go

```diff
@@ -294,10 +301,10 @@ func handleUpdateInstallation(c *Context, w http.ResponseWriter, r *http.Request
 		return
 	}
 
-	if patchInstallationRequest.Apply(installation) {
+	if patchInstallationRequest.Apply(installation.Installation) {
```

Comment:
With the change of this `installation` variable to a `DTO` object, its name should change as well (as mentioned in another comment on the changing of a function definition for `lockCluster`) to `installationDTO` or something similar, to reflect the change in its type, and also to prevent the seemingly-nonsensical "variable `installation` with value `Installation` reference above, as well as the stutter ("`thing Thing`").